src/main/java/dang/gun/com/image/ImageService.java

```diff
@@ -39,6 +39,31 @@ public void save(List<MultipartFile> fileList, int postId) throws IOException {
         }
     }
 
+    /**
+     * 이미지 수정
+     *
+     * @param fileList
+     * @param postId
+     * @throws IOException
+     */
+    public void update(List<MultipartFile> fileList, int postId) throws IOException {
+        for (int i = 0; i < fileList.size(); i++) {
```

Comment:
i가 꼭 필요한게 아니라면 아래 같은 방법도 있습니다!
```suggestion
        for (MultipartFile file : fileList) {
```

Reply:
감사합니다!! 저 방법을 알고있으면서 잘 안쓰게되네요,,,까먹어서,,,